ca.uwaterloo.Raven.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "ca.uwaterloo.Raven",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "Raven.exe",
+  "finish-args":
+  [
+    "--filesystem=home"
```

Comment:
so this display nothing?

Reply:
If I understand correctly: yes - this is a scientific software application that is primarily used from the command line.